generateInterpData.m

```diff
@@ -159,7 +159,7 @@
     
     % Remove any data which is associated with vertices that are not 
     % referenced by the triangulation
-    [~,isVertUsed] = repack(getMesh(userdata));
```

Comment:
suggest use getMesh() here rather than accessing userdata.surface.triRep directly; and pass the parameter/value 'type' to ensure that a triangulation is returned:

getMesh(userdata, 'type', 'triangulation')